test/public/createUploadLink.test.js

```diff
@@ -654,6 +656,22 @@ module.exports = (tests) => {
     }
   );
 
+  tests.add('`createUploadLink` with `execute`.', async () => {
+    const link = createUploadLink({
+      fetch: async () =>
+        new Response(
+          JSON.stringify({ data: { a: true } }),
+          graphqlResponseOptionsOk
+        ),
+    });
+    const query = '{\n  a\n}\n';
+    const result = await toPromise(execute(link, { query: gql(query) }));
+
+    deepStrictEqual(result, {
+      data: { a: true },
+    });
+  });
```

Comment:
I see that you have also removed the `fetchUri` and `fetchOptions` assertions; don’t you think we should check that fetching works as intended, like how it’s done in the other tests?

Reply:
It seemed extraneous, but I can add it back in.